src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -304,13 +313,16 @@ private void MarkBasicBlock(BasicBlock basicBlock)
 
         private void ImportBasicBlock(BasicBlock basicBlock)
         {
-            _stackTop = 0;
+            _stack.Clear();
```

Comment:
ILImporter is intentionally factored to be reusable lowest level part of IL reader. It is being reused in multiple different projects (e.g. https://github.com/Microsoft/binskim/pull/10/commits/3977498ad04cf2013fa869dd5a711f2d7f0ad8d4).

It is common for each project to make tweaks to it (that are incompatible with other uses). Building abstract class or assembly out of it would kill its reusability because of these custom tweaks would not be to do possible anymore.